core/include/ignition/plugin/Plugin.hh

```diff
@@ -137,6 +137,10 @@ namespace ignition
       public: bool HasInterface(const std::string &_interfaceName,
                                 const bool _demangled = true) const;
 
+      /// \brief Returns the name of the plugin, or nullptr if the plugin is not
+      /// instantiated.
```

Comment:
Mind documenting the `\return`?

Reply:
Done. Updated doc of HasInterface above as well. It was the culprit here. Copy-pasting numbs your brain